js/webform_civicrm_admin.js

```diff
@@ -541,16 +541,20 @@ var wfCiviAdmin = (function (D, $, once) {
         }
       }).change();
 
-      function billingMessages() {
+      function billingMessages(contactId) {
```

Comment:
Is this param required @rubofvil ? I dont see it being used on any call.

Reply:
This isn't required, we cleaned in the last rebase, thx